lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1426,90 +1436,55 @@ def _display_shutdown_message():
         d.addCallback(lambda _: reactor.callLater(0.0, reactor.stop))
         return self._render_response("Shutting down")
 
-    def jsonrpc_get_lbry_files(self):
-        """
-        DEPRECATED. Use `file_list` instead.
-        """
-        return self.jsonrpc_file_list()
-
-    def jsonrpc_file_list(self):
+    @defer.inlineCallbacks
+    def jsonrpc_file_list(self, **kwargs):
         """
-        List files
+        List files limited by optional filters
 
         Args:
-            None
-        Returns:
-            List of files, with the following keys:
-            'completed': bool
-            'file_name': string
-            'key': hex string
-            'points_paid': float
-            'stopped': bool
-            'stream_hash': base 58 string
-            'stream_name': string
-            'suggested_file_name': string
-            'sd_hash': string
-        """
-
-        d = self._get_lbry_files()
-        d.addCallback(lambda r: self._render_response([d[1] for d in r if d[0]]))
-
-        return d
-
-    def jsonrpc_get_lbry_file(self, **kwargs):
-        """
-        DEPRECATED. Use `file_get` instead.
-        """
-        return self.jsonrpc_file_get(**kwargs)
-
-    def jsonrpc_file_get(self, **kwargs):
-        """
-        Get a file, if no matching file exists returns False
+            'name' (optional): filter files by lbry name,
+            'sd_hash' (optional): filter files by sd hash,
+            'file_name' (optional): filter files by the name in the downloads folder,
+            'stream_hash' (optional): filter files by stream hash,
+            'claim_id' (optional): filter files by claim id,
+            'outpoint' (optional): filter files by claim outpoint,
+            'rowid' (optional): filter files by internal row id,
+            'full_status': (optional): bool, if true populate the 'message' and 'size' fields
 
```

Comment:
not sure why this full_status option is necessary. Why not just always return full status